### PR TITLE
Reader: Add Conversations sidebar item

### PR DESCRIPTION
**Pull request by blowery**
screenshot 2017-07-06 15 53 07

Links to a blank placeholder for now. Only appears if the feature flag is active.

### CODE REVIEW

**Comment by blowery** [comment]
This will fail tests due to lint errors on the sidebar item.

File: client/reader/conversations/controller.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { recordTrack } from 'reader/stats';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import AsyncLoad from 'components/async-load';
+import { trackPageLoad } from 'reader/controller-helper';
+
+const ANALYTICS_PAGE_TITLE = 'Reader';
+
+export function conversations( context ) {
+	const basePath = route.sectionify( context.path );
+	const fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Conversations';
```
Comment (samouri) [inline client/reader/conversations/controller.js]: what the point of this convention.

can we just make a locally scoped `analyticsPageTitle = Reader > Conversations`?

**Comment by samouri** [comment]
not a question specific to this PR in particular, but whats our current logic for when to/not to split something out at as an async chunk? 

My thought here is that it could be better for this to be part of the reader bundle

File: client/reader/conversations/controller.js (hunk 1 above)
Comment (blowery) [inline client/reader/conversations/controller.js]: done

**Comment by blowery** [comment]
i've been splitting off every major portion. It could be better to keep this internal, but for now I'd like to keep with the existing pattern and not impact the download size for following.

**Pull request merged**